src/MaxText/examples/README.md

```diff
@@ -128,6 +128,29 @@ Use the link for Jupyter Lab as a link for "Connect to a local runtime" in Colla
 ### GRPO Training
 
 - **`grpo_llama3_demo.ipynb`** → GRPO training on math dataset
```

Comment:
Since you are using #### for the python script, maybe put `#### GRPO colab usage` here, and can we call it `grpo_llama3_1_8b_demo.ipynb`

Reply:
Yes put there